### PR TITLE
Switch to yarn

### PR DESCRIPTION
**Pull request by PVince81**
Removed bower.json and handlebars deps as the version from core is
already loaded in the browser.

Fixes https://github.com/owncloud/customgroups/issues/121

### CODE REVIEW

**Comment by PVince81** [comment]
@DeepDiver1975 all green now

**Comment by PVince81** [comment]
@DeepDiver1975 please review

**Comment by PVince81** [comment]
@DeepDiver1975 do we still want this or close ?

**Comment by PVince81** [comment]
@DeepDiver1975 please review

File: templates/index.php [hunk 1]
```diff
@@ -23,7 +23,6 @@
 style('settings', 'settings');
 script('core', 'oc-backbone');
 script('core', 'oc-backbone-webdav');
-script('customgroups', 'vendor/handlebars/handlebars');
```
Comment (DeepDiver1975) [inline templates/index.php]: so you are relying on core's handlebars ... fine by me

**Pull request merged**